c

```diff
@@ -0,0 +1,341 @@
+Commencing CoreCLR Repo build
```

Comment:
I assume this was added by mistake.

Reply:
Yup it was supposed to be the diff on stdout, somehow got persisted in the source tree. Removed.